xDSCResourceDesigner.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.4.0.0'
+ModuleVersion = '1.4.0.1'
```

Comment:
versions must correspond to releases in the PSGallery. It also would help reduce merge conflicts among other contributors.